tests/integration/runner/run.php

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env php
+<?php
+
+require_once __DIR__ . '/../../../vendor/autoload.php';
+require_once __DIR__ . '/util.php';
+require_once __DIR__ . '/FolderComparator.php';
+
+use PrestaShop\HeaderStamp\Command\UpdateLicensesCommand;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\Filesystem\Filesystem;
+
+$modulesToTest = [
+    'gsitemap' => [],
+];
+$workspaceID = 100;
+$filesystem = new Filesystem();
+$folderComparator = new FolderComparator();
+
+$application = buildTestApplication();
+
+foreach ($modulesToTest as $moduleName => $config) {
+    ++$workspaceID;
+    $moduleFolderpath = __DIR__ . '/../module-samples/' . $moduleName;
+    $expectedModuleFolderpath = __DIR__ . '/../expected/' . $moduleName;
+    $workspaceFolderpath = __DIR__ . '/../workspace/' . $workspaceID;
```

Comment:
Do you think we should ignore this folder?

Reply:
Yes we should. I put it in my .gitignore at some point but it seems I got it wrong and it was not pushed

.gitignore should contain tests/integration/workspace/* (but the .gitkeep inside it should be into git)

Reply:
Fixed in https://github.com/PrestaShopCorp/header-stamp/pull/19/commits/9e5a9d5fbcd7a60fcf6cddcc1806f85dd7448d79